fetch.bs

```diff
@@ -264,6 +270,69 @@ use "<code>deprecated</code>" is not defined by this specification. An
 <a>environment settings object</a> typically derives its
 <a for="environment settings object">HTTPS state</a> from a <a for=/>response</a>.
 
+<p>To
+<dfn export lt="collect an HTTP quoted string|collecting an HTTP quoted string">collect an HTTP quoted string</dfn>
+from a <a for=/>string</a> <var>input</var>, given a <a>position variable</a> <var>position</var>,
+run these steps:
+
+<ol>
+ <li><p>Let <var>positionStart</var> be <var>position</var>.
+
+ <li><p>Let <var>value</var> be the empty string.
+
+ <li><p>Assert: the <a>code point</a> at <var>position</var> within <var>input</var> is U+0022 (").
+
+ <li><p>Advance <var>position</var> by 1.
+
+ <li>
+  <p>While true:
+
+  <ol>
+   <li><p>Append the result of <a>collecting a sequence of code points</a> that are not U+0022 (")
+   or U+005C (\) from <var>input</var>, given <var>position</var>, to <var>value</var>.
+
+   <li>
+    <p>If <var>position</var> is not past the end of <var>input</var>, then:
```

Comment:
It would be simpler to flip this so that you break if position is past the end of input. That would probably also allow you to remove the "continue" step later.

Reply:
I added a commit that does this, but note that you'll need an additional break due to having to deal with ". But there's less nesting, so probably acceptable.